mirgecom/artificial_viscosity.py

```diff
@@ -158,9 +160,11 @@ class _AVRTag:
 
 
 def av_laplacian_operator(discr, boundaries, fluid_state, alpha, gas_model=None,
```

Comment:
```suggestion
def av_laplacian_operator(dcoll, boundaries, fluid_state, alpha, gas_model=None,
```
(Here and elsewhere.)